authors/apps/authentication/models.py

```diff
@@ -77,6 +78,18 @@ class User(AbstractBaseUser, PermissionsMixin, TimeStampedModel):
     # falsed.
     is_staff = models.BooleanField(default=False)
 
+    # A timestamp representing when this object was created.
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    # A timestamp reprensenting when this object was last updated.
+    updated_at = models.DateTimeField(auto_now=True)
+
+    # Social authetication id field
+    social_auth_id = models.CharField(
+        db_index=False, max_length=255,
+        default=create_unique_social_id_number
+        )
```

Comment:
Please align this paren with `social_auth_id`. This is so that your code can be consistent with our current project conventions.